easybuild/easyconfigs/l/libdrm/libdrm-2.4.99-GCCcore-8.3.0.eb

```diff
@@ -19,7 +19,7 @@ builddependencies = [
 dependencies = [('X11', '20190717')]
 
 sanity_check_paths = {
-    'files': ['include/xf86drm.h', 'include/xf86drmMode.h', 'lib/libdrm_intel.%s' % SHLIB_EXT,
```

Comment:
@edmondac Can you clarify why you're removing this?

It should be done in a separate PR (consistently in the recent `libdrm` easyconfigs, and with some motivation)

Reply:
Done in https://github.com/easybuilders/easybuild-easyconfigs/pull/9694